gitops/addons/charts/gitops-bridge/values.yaml

```
@@ -768,14 +770,45 @@ addons:
         - key: enable_grafana_operator
           operator: In
           values: ['true']
-
     resources:
       path: charts/resources/grafana-operator
       helm:
         valuesObject:
           AMP_ENDPOINT_URL: '{{.metadata.annotations.amp_endpoint_url}}'
           AMP_AWS_REGION: '{{.metadata.annotations.aws_region}}'
-
+  opentelemetry_operator:
+    enabled: false
+    releaseName: opentelemetry-operator
+    namespace: '{{default "opentelemetry-operator-system" (index .metadata.annotations "opentelemetry_operator_namespace")}}'
```

```suggestion
    namespace: "opentelemetry-operator-system"
```
operator doesn't need access to AWS, it can be a simple helm chart with default namespace